directorylisting.yaml

```diff
@@ -1,5 +1,6 @@
 enabled: true
 level: 1
+max-depth: 3
```

Comment:
This needs to be `max_depth` with the underscore rather than the dash, as used elsewhere, or the setting won't be picked up.

Reply:
Good catch, thank you. Corrected.